ipyleaflet/leaflet.py

```diff
@@ -1866,38 +1870,6 @@ def __init__(self, legend, *args, name="Legend", **kwargs):
         self.title = name
         self.legend = legend
 
-    @property
-    def name(self):
-        return self.title
-
-    @name.setter
-    def name(self, title):
-        self.title = title
-
-    @property
-    def legends(self):
-        return self.legend
-
-    @legends.setter
-    def legends(self, legends):
-        self.legend = legends
-
-    @property
-    def positioning(self):
-        return self.position
-
-    @positioning.setter
-    def positioning(self, position):
-        self.position = position
-
-    @property
-    def positionning(self):
-        return self.position
-
-    @positionning.setter
-    def positionning(self, position):
-        self.position = position
-
```

Comment:
This part should probably be done in a separate PR?

Reply:
Ok, this change has been removed and will be included in a separate PR.